modules/authentication/package.json

```diff
@@ -41,6 +41,7 @@
     "jwks-rsa": "^3.1.0",
     "lodash-es": "^4.17.21",
     "moment": "^2.30.1",
+    "otp-generator": "^4.0.1",
```

Comment:
we don't need a library for this. Write a utility function to generate a requested number of digits.